aspnet/dnx/commands.rst

```diff
@@ -1,8 +1,232 @@
+        "Microsoft.AspNet.Hosting": "1.0.0-beta6",
+        "Microsoft.AspNet.Server.IIS": "1.0.0-beta6",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-beta6",
+        "Microsoft.AspNet.StaticFiles": "1.0.0-beta6"
+      },
+      "commands": {
+		"web": "Microsoft.AspNet.Hosting --config hosting.ini",
+		"kestrel": "Microsoft.AspNet.Hosting --config hosting.ini",
+		"gen": "Microsoft.Framework.CodeGeneration",
+		"ef": "EntityFramework.Commands"
+      },
+      "frameworks": {
+        "dnx451": { },
+        "dnxcore50": { }
+      }
+    }
+
+A command can include a set of arguments that will be passed to the DNX. In the above example, the first part of a command statement is an assembly with an entry point that the DNX will try to execute. Notice that in the ``commands`` section shown above, the ``ef`` command is implemented by the ``EntityFramework.Commands`` assembly. This command doesn't require any extra argument, all that is needed to define the command is the name of the assembly. For the ``web`` command and the ``kestrel`` command, the arguments are contained in the referenced *hosting.ini* file. In the *hosting.ini* file you will see the ``server`` agrument and the the ``server.urls`` argument. The ``kestrel`` command, as well as the ``web`` command, will check the ``Microsoft.AspNet.Hosting`` assembly for an entry point, then it will pass the ``server`` and ``serveral.urls`` agruments to the entry point. 
+
+.. note:: The assembly listed in the ``commands`` section should be pulled in by a package that your application depends on.
+
+Running commands using dnx.exe
+------------------------------
+
+You can use DNX to run the commands defined by your project by entering the following in the command prompt from your project's directory::
+
+	dnx [command]
+
+You can also run commands from VS Code or Visual Studio. From VS Code, open the **Command Palette** (Ctrl+Shift+P) and enter the name of the command you want to run. From Visual Studio, open the **Command Window** (Ctrl+Alt+A) and enter the name of the command you want to run.
+	
+For example, the following command is used to run a web application::
+
+    dnx kestrel
+
+To run a console app, you can use the following command::
+
+	dnx run
+	
+.. note:: Note that running a command is short-hand for specifying the command assembly and it's arguments directly to DNX. For example, ``dnx web`` is a short-hand alias for ``dnx Microsoft.AspNet.Hosting hosting.ini``, where the hosting.ini file contains the command parameters. 
+
+Global commands
+---------------
+Global commands are DNX console applications (in a NuGet package) that are installed globally and runnable from your command line. Global commands are different from the commands that your add in the ``commands`` section of the *project.json* file of a project because global commands are not project specific. You can install, run, uninstall, build, and publish global commands. 
```

Comment:
Um... yes and no. Global commands are different but they are the same. A global command is just a project command that's installed globally. So, in other words, it is just a command that's made available to everything that runs under that user profile.